server/config/routes.js

```diff
@@ -0,0 +1,46 @@
+var controller = require('../controllers');
+var request = require('request');
+
+
+module.exports = function(app) {
+
+  app.get('/', function (req, res) {
+
+  });
+
+  // app.post('/api/signin', );
+  // app.post('/api/signup', );
+  // app.get('/api/recent', );
+  
+  app.get('/api/stories', function (req, res) {
+    controller.stories.getAllStories(req, res);
```

Comment:
They way you have it now, you can pass controller.stories.getAllStories directly to app.get.
```
app.get('/api/stories', controllers.stories.getAllStories);
```
but the controller function probably shouldn't take request and response objects as parameters. They should take more primitive parameters. In this case it doesn't take any parameters I think.